src/components/Sidebar.jsx

```diff
@@ -17,59 +18,50 @@ import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 import flag from 'cozy-flags'
 import { routes } from 'constants/routes'
 
+const makeClassName = () => ({
```

Comment:
nit: why not using a constant?
```suggestion
const makeClassName = ({
```

Reply:
Needs to be a function because we have to execute it each render. I'm not a fan of the implementation because it's a lot of copy-pasted calls but I will refactor all that in my next ticket where I work on the menu

Reply:
ok  thanks